src/Microsoft.Security.Utilities.Core/IdentifiableScan.cs

```diff
@@ -123,7 +125,19 @@ public void Start()
     {
         if (this.scan == IntPtr.Zero)
         {
-            this.scan = identifiable_scan_create();
+            var builder = new StringBuilder();
+
+            foreach (var id in this.idToLengthMap.Keys)
+            {
+                builder.Append(id);
+		builder.Append(';');
```

Comment:
>builder [](http://example.com/codeflow?start=2&length=7)

nit, our convention for dropping lines on parameters is for the first arg to remain on the same line as the call, and args to left-align underneath that location.

an old Windows C++ convention i came to like. :)

Reply:
Fixed to just a single line, since only 2 args.